packages/asciidoctor/index.d.ts

```diff
@@ -0,0 +1,19 @@
+export interface IRuntime {
```

Comment:
Is it possible to add any kind of tests to make sure that the syntax is valid and that the definition matches the API?

Reply:
Yes, I think I can write typescript tests for this functionality